greentest/2.7/test_ssl.py

```diff
@@ -54,6 +61,17 @@ def test_sslwrap_simple(self):
 # Issue #9415: Ubuntu hijacks their OpenSSL and forcefully disables SSLv2
 def skip_if_broken_ubuntu_ssl(func):
     if hasattr(ssl, 'PROTOCOL_SSLv2'):
+        if hasattr(ssl, 'SSLContext'): # >= 2.7.9
+            def f(*args, **kwargs):
+                try:
+                    ssl.SSLContext(ssl.PROTOCOL_SSLv2)
+                except ssl.SSLError:
+                    if (ssl.OPENSSL_VERSION_INFO == (0, 9, 8, 15, 15) and
```

Comment:
I see where this came from, but I am not sure why it was brought over in the first place. Looks like the origin was just a hack for a specific build farm setup.

Would be nice if we can drop this wart if no one is actually going to hit this.